person.rb

```diff
@@ -1,4 +1,4 @@
-require './nameable'
+require_relative './nameable'
 
 class Person < Nameable
```

Comment:
#### As requested by the previous reviewer
- [ ] This person class needs to have a `@rentals` instance variable initialized to an empty array within the initialize method, like so:

  ```
   def initialize(age, name = 'Unknown', parent_permission: true)
    super()
    @id = Random.rand(1..100)
    @age = age
    @name = name
    @parent_permission = parent_permission
    @rentals = []
  end
  ```

- [ ] You need to assign `attr_accessor` to this `:rentals`, just as you have done `:name` and `:age`:
  ```
  attr_accessor :name, :age, :rentals
  ```
- [ ] You need to create the `add_rental` method for this person class as well:
  ```
  def add_rental(date, book)
    Rental.new(date, self, book)
  end
  ```